code/modules/materials/_materials.dm

```diff
@@ -868,7 +868,7 @@ INITIALIZE_IMMEDIATE(/obj/effect/gas_overlay)
 		holder.remove_reagent(type, REAGENT_VOLUME(holder, type))
 		. = TRUE
 
-/decl/material/proc/affect_overdose(var/mob/living/M) // Overdose effect. Doesn't happen instantly.
+/decl/material/proc/affect_overdose(mob/living/M, total_dose) // Overdose effect. Doesn't happen instantly.
```

Comment:
is `tick_dosage_tracker` a private variable or something? how is total_dose better than just `M.tick_dosage_tracker[src]`?